sample-apps/grpc-greeter-go/greeter-istio.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: greeter
+spec:
+  selector:
+    istio: ilbgateway
```

Comment:
This sample app will live in the generic `sample-apps/` directory, not coupled with an ILB tutorial. I wonder if having ILBGateway as default here is a good idea, vs. `ingressgateway`.

Reply:
Good point. I've added two gateway manifests, one for Ingress Gateway and one for ILB Gateway, so users can choose which one to apply.